### PR TITLE
Release flow improvements

### PR DESCRIPTION
**Pull request by nachoiacovino**
Fixes FTD-772, FTD-676, FTD-771 and FTD-713

### CODE REVIEW

File: components/contract-components/contract-release-form/index.tsx [hunk 1]
```diff
@@ -214,11 +224,28 @@ export const ContractReleaseForm: React.FC<ContractReleaseFormProps> = ({
...
+                  value={logoUrl}
+                  setValue={(file) => setValue("logo", file)}
+                  border="1px solid"
+                  borderColor="gray.200"
+                  borderRadius="md"
+                  transition="all 200ms ease"
+                  _hover={{ shadow: "sm" }}
+                  renderPreview={(fileUrl) => (
+                    <MaskedAvatar w="100%" h="100%" src={fileUrl} />
+                  )}
```
Comment (jnsdls) [inline components/contract-components/contract-release-form/index.tsx:240]: @nachoiacovino why would the contract image be a hexagon?

File: components/pages/release.tsx [hunk 2]
```diff
@@ -59,16 +61,29 @@ export const ReleaseWithVersionPage: React.FC<ReleaseWithVersionPageProps> = ({
...
-            alt=""
-          />
+          {release?.logo ? (
+            <MaskedAvatar
+              src={release.logo.replace(
+                "ipfs://",
+                `${StorageSingleton.gatewayUrl}/`,
+              )}
+              boxSize={14}
+            />
```
Comment (jnsdls) [inline components/pages/release.tsx:77]: why do releases have hex images?

**Review by jnsdls — CHANGES_REQUESTED** [review]
I don't think we want release logos to be hexagons

File: components/pages/release.tsx (hunk 2 above)
Comment (nachoiacovino) [reply components/pages/release.tsx]: no reason I guess, changed

**Review by jnsdls — APPROVED** [review]
LGTM

**Pull request merged**